lib/globalize_uuid.rb

```diff
@@ -0,0 +1,19 @@
+module Globalize
+  module ActiveRecord
+    module Migration
+      class Migrator
+        def primary_key_type
+          column_type(model.primary_key).to_sym
+        end
+
+        def create_translation_table
+          connection.create_table(translations_table_name, id: primary_key_type) do |t|
+            t.references table_name.sub(/^#{table_name_prefix}/, '').singularize, null: false, type: primary_key_type
```

Comment:
Line is too long. [117/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.